### ISSUE TITLE
a11y: the colour of the TOC does not meet colour contrast

### ISSUE THREAD

**Issue by schalkneethling** [github/docs#21218]
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

All articles with a TOC.

### What changes are you suggesting?

Recently, I think in the last two weeks, the colour of the links in the "In this article" section has changed to light gray. This does not meet WCAG guidelines for colour contrast and is hard to read, as seen in the screenshot below.

I wonder if this is because of a chance in Primer?

![Screenshot showing low contrast links under the, "In this article" table of contents sidebar](https://user-images.githubusercontent.com/10350960/194727039-4c282c29-30d4-4aa5-bad7-1045e359b16b.png)

**Comment by cmwilson21** [comment]
👋 @schalkneethling - Thanks so much for opening an issue! I'll triage this for the team to take a look :eyes:

**Comment by rsese** [comment]
We do believe it's related to Primer 👍  We have an internal issue tracking this one so I'll go ahead and close this out but thanks again for the heads up!

(fwiw, we found this happens for folks that have their OS settings set to dark mode but their github.com settings set to light mode)

**Comment by schalkneethling** [comment]
@rsese and @cmwilson21 wanted to highlight another instance of a color contrast problem. This instance is not a TOC.
https://docs.github.com/en/rest/repos#edit

![Screenshot showing some of the text in a notice block being illegible.](https://user-images.githubusercontent.com/10350960/197416021-2a899086-741f-45c5-8b01-9996ebe5e9cc.png)

**Comment by cmwilson21** [comment]
@schalkneethling Thanks for bringing this one up, too. I'll add it to our internal issue.

We appreciate you catching and reporting these instances! ✨

**Comment by peterbe** [comment]
Hi @schalkneethling!!
The actual bug is here: https://github.com/primer/react/issues/2229 which is also a public project.

**Comment by schalkneethling** [comment]
> Hi @schalkneethling!! The actual bug is here: [primer/react#2229](https://github.com/primer/react/issues/2229) which is also a public project.

Thanks!
